chain/network/src/peer.rs

```diff
@@ -255,7 +256,15 @@ impl Peer {
                 #[cfg(feature = "metric_recorder")]
                 self.peer_manager_addr.do_send(metadata.set_size(bytes.len()));
                 self.tracker.increment_sent(bytes.len() as u64);
-                self.framed.write(bytes);
+                let bytes_len = bytes.len();
+                if !self.framed.write(bytes) {
+                    error!(
+                        "{} Failed to send message {} of size {}",
+                        get_tid(),
+                        format!("routed message from {}", strum::AsStaticRef::as_static(msg)),
```

Comment:
the message we send here is not necessarily routed

Reply:
fixed